ktor-server/ktor-server-plugins/ktor-server-pebble/jvm/src/io/ktor/server/pebble/Pebble.kt

```diff
@@ -38,14 +48,19 @@ public class PebbleContent(
  * Provides the ability to respond with [PebbleContent].
  * You can learn more from [Pebble](https://ktor.io/docs/pebble.html).
  */
-public val Pebble: ApplicationPlugin<PebbleEngine.Builder> = createApplicationPlugin(
-    "Pebble",
-    { PebbleEngine.Builder() }
-) {
+public val Pebble: ApplicationPlugin<PebbleConfiguration> = createApplicationPlugin("Pebble", ::PebbleConfiguration) {
     val engine = pluginConfig.build()
 
-    fun process(content: PebbleContent): OutgoingContent = with(content) {
+    fun process(content: PebbleContent, call: ApplicationCall): OutgoingContent = with(content) {
         val writer = StringWriter()
+        val availableLanguages: List<String>? = pluginConfig.availableLanguages?.toList()
```

Comment:
please extract it to `createApplicationPlugin` to save the reference at install time to ignore possible mutations of config later